app/src/pages/playground/SpanPlaygroundPage.tsx

```diff
@@ -1,5 +1,8 @@
-import React, { useMemo } from "react";
-import { useLoaderData } from "react-router";
+import React, { useMemo, useState } from "react";
+import { useLoaderData, useNavigate } from "react-router";
+import { divide } from "lodash";
```

Comment:
start typing, press tab, get a lodash import

this is the way of the world

Reply:
haha, yeah this happens to me so much